schema/objects/document-state.schema.yaml

```diff
@@ -4,3 +4,7 @@ description:
   and will see additions in future document versions.
 type: object
 additionalProperties: true
+optional:
+  - patchInfo
+properties:
+  patchInfo: { $ref: ./patch-info.schema.yaml }
```

Comment:
Sorry @jedrichards I didn't spot this before and it probably wasn't clear from our conversations, but `patchInfo` is in fact a property on the main document data rather than on `documentState`.

Reply:
Oh ok!